doc/integrator/checker.rst

```diff
@@ -50,6 +50,24 @@ The checker use the following configuration structure:
         checker:
             forward_headers: ['Cookie', 'Authorisation']
 
+.. note::
+
+    The checker assumes that it can access the c2cgeoportal services via ``http://localhost``.
+    If this is not allowed on your server, you can override this behaviour as follows.
+    In the ``vars`` - ``checker`` configuration (see above), add the following:
+
+    .. code:: yaml
+
+            rewrite_as_http_localhost: False
+
+    Now, in your project YAML configuration, instead of defining the ``checker_path``,
+    define a `checker_url` with the full URL to be used, for example:
+
+    .. code:: yaml
+
+        checker_url: https://${host}/${instanceid}/wsgi/check_collector?
```

Comment:
same

Reply:
here it is a bit more difficult to show, because it is "top-level" in the project yaml file (not the vars file). But I will make a new proposal.